diff -dur pragha-0.97.0.orig/configure pragha-0.97.0/configure
--- pragha-0.97.0.orig/configure	2011-07-11 00:47:23.000000000 +0200
+++ pragha-0.97.0/configure	2011-07-30 21:18:31.280122074 +0200
@@ -2854,13 +2854,13 @@
 	   yes)	debug=true ;
 		CFLAGS="$CFLAGS -DDEBUG -D_DEBUG -g -ggdb -O0 -fno-inline -fno-inline-functions" ;;
 	    no)	debug=false ;
-		CFLAGS="$CFLAGS -DNDEBUG -O3 -Werror" ;;
+		CFLAGS="$CFLAGS -DNDEBUG -Werror" ;;
 	     *)	as_fn_error $? "bad value ${enableval} for --enable-debug" "$LINENO" 5 ;;
 	esac
 
 else
   	debug=false
-	CFLAGS="$CFLAGS -DNDEBUG -O3 -Werror"
+	CFLAGS="$CFLAGS -DNDEBUG -Werror"
 
 fi
 
diff -dur pragha-0.97.0.orig/configure.ac pragha-0.97.0/configure.ac
--- pragha-0.97.0.orig/configure.ac	2011-07-03 23:10:54.000000000 +0200
+++ pragha-0.97.0/configure.ac	2011-07-30 21:18:55.708262113 +0200
@@ -29,11 +29,11 @@
 	   yes)	debug=true ;
 		CFLAGS="$CFLAGS -DDEBUG -D_DEBUG -g -ggdb -O0 -fno-inline -fno-inline-functions" ;;
 	    no)	debug=false ;
-		CFLAGS="$CFLAGS -DNDEBUG -O3 -Werror" ;;
+		CFLAGS="$CFLAGS -DNDEBUG -Werror" ;;
 	     *)	AC_MSG_ERROR(bad value ${enableval} for --enable-debug) ;;
 	esac
 ],[	debug=false
-	CFLAGS="$CFLAGS -DNDEBUG -O3 -Werror"
+	CFLAGS="$CFLAGS -DNDEBUG -Werror"
 ])
 AC_SUBST(CFLAGS)
 
